pkg/k8shandler/kibana/reconciler.go

```diff
@@ -363,6 +364,12 @@ func isDeploymentDifferent(current *apps.Deployment, desired *apps.Deployment) (
 		different = true
 	}
 
+	if current.Spec.Replicas != desired.Spec.Replicas {
+		logrus.Debugf("Replicas changed. Previous: %d, Current: %d", current.Spec.Replicas, desired.Spec.Replicas)
```

Comment:
Please add the "kibana" instance name to the debug line given we can/could support multiples

Reply:
fixed